IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -410,8 +410,13 @@ - (BOOL)removeItemsWithKey:(MSIDCacheKey *)key
                    context:(id<MSIDRequestContext>)context
                      error:(NSError **)error
 {
-    MSID_LOG_NO_PII(MSIDLogLevelVerbose, nil, context, @"Remove keychain items, key info (account: %@ service: %@, keychainGroup: %@)", _PII_NULLIFY(key.account), _PII_NULLIFY(key.service), [self keychainGroupLoggingName]);
-    MSID_LOG_PII(MSIDLogLevelVerbose, nil, context, @"Remove keychain items, key info (account: %@ service: %@, keychainGroup: %@)", key.account, key.service, self.keychainGroup);
+    NSString *account = key.account;
+    NSString *service = key.service;
+    NSData *generic = key.generic;
+    NSNumber *type = key.type;
+    
+    MSID_LOG_NO_PII(MSIDLogLevelVerbose, nil, context, @"Remove keychain items, key info (account: %@ service: %@, keychainGroup: %@)", _PII_NULLIFY(account), _PII_NULLIFY(service), [self keychainGroupLoggingName]);
```

Comment:
nit: is there any PII in service? If not, should we remove it?

Reply:
Client id is there in service which I think is PII.